### ISSUE TITLE
Edit LL15 with the edit_apsimx function 

### ISSUE THREAD

**Issue by SumanGajurel** [femiguez/apsimx#96]
The edit_apsimx function doesnt work with LL15 but works fine with other soil nodes such as BD, DUL. Could you look into that please?

**Comment by femiguez** [comment]
Thanks for the report @SumanGajurel ! I have fixed this now. In the meantime, this works

```
## Testing editing LL15
library(apsimx)
apsimx_options(warn.versions = FALSE)

ex.dir <- auto_detect_apsimx_examples()

tmpd <- tempdir()
setwd(tmpd)
file.copy(file.path(ex.dir, "Maize.apsimx"), ".")

pp <- inspect_apsimx("Maize.apsimx", 
                     node = "Soil",
                     soil.child = "Physical",
                     parm = "LL15",
                     print.path = TRUE)

ll15 <- rep(0.28, 7)

edit_apsimx("Maize.apsimx", 
            node = "Other",
            parm.path = pp,
            value = ll15,
            overwrite = TRUE)

inspect_apsimx("Maize.apsimx", 
               node = "Soil",
               soil.child = "Physical",
               parm = "LL15")
```

**Comment by SumanGajurel** [comment]
@femiguez thank you for fixing this but it doesn't work this way,
edit_apsimx("Siding.apsimx", src.dir = ".", node = "Soil", soil.child = "Physical", parm = "LL15", value = rep(1, 7), overwrite = T)

**Comment by femiguez** [comment]
It is fixed in the latest version from github (2.3.5). You can install it using the remotes package. Just tested it.

```
## Testing editing LL15
library(apsimx)
packageVersion("apsimx")
apsimx_options(warn.versions = FALSE)

ex.dir <- auto_detect_apsimx_examples()

tmpd <- tempdir()
setwd(tmpd)
file.copy(file.path(ex.dir, "Maize.apsimx"), ".")

#### Method 1 ####
pp <- inspect_apsimx("Maize.apsimx", 
                     node = "Soil",
                     soil.child = "Physical",
                     parm = "LL15",
                     print.path = TRUE)

ll15 <- rep(0.28, 7)

edit_apsimx("Maize.apsimx", 
            node = "Other",
            parm.path = pp,
            value = ll15,
            overwrite = TRUE)

inspect_apsimx("Maize.apsimx", 
               node = "Soil",
               soil.child = "Physical",
               parm = "LL15")


#### Method 2 ####
edit_apsimx("Maize.apsimx", 
            node = "Soil",
            soil.child = "Water",
            parm = "LL15",
            value = rep(0.29, 7),
            overwrite = TRUE)

inspect_apsimx("Maize.apsimx", 
               node = "Soil",
               soil.child = "Physical",
               parm = "LL15")
```

**Comment by SumanGajurel** [comment]
Thank you so much. It works now.
